core/metamodel/src/main/java/org/apache/isis/core/metamodel/specloader/SpecificationCacheDefault.java

```diff
@@ -100,7 +102,9 @@ public void forEach(Consumer<T> onSpec, boolean shouldRunConcurrent) {
         if(shouldRunConcurrent) {
             vList.forEachParallel(onSpec);
         } else {
-            vList.forEach(onSpec);
+            vList
+                .stream().sorted(Comparator.comparing(HasLogicalType::getLogicalTypeName))
```

Comment:
Is there a need for sorting? Or is it a debugging thing.

Reply:
It's a presentation thing in the graphQL metamodel - seems reasonable behaviour?